_includes/block/head.html

```diff
@@ -1,43 +1,33 @@
 <head>
   <meta charset="utf-8">
-  
-  <title>{{ site.name }}{% if page.title %} - {{ page.title }}{% endif %}</title>
-  <meta name="description" content="{{site.description}}">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <!-- Compiled and minified CSS -->
-  <link href="https://fonts.googleapis.com/css?family=Raleway:400,400i,800,900" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">  
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+  <title>{% if page.title %}{{ page.title }} — {% endif %}{{ site.name }}</title>
+  <meta name="description" content="{{ site.description }}">
 
-  <link rel="stylesheet" href="{{ '/assets/css/main.css' | relative_url }}" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
should we leverage `preload` for these assets? (speed up first time visits)

Reply:
I would personally lean toward no, just to err on the side of caution. According to the MDN page @chrisvogt linked to, it's still in Candidate Recommendation status, so only being experimentally implemented now, with no support in Internet Explorer yet. And according to [the W3C document](https://w3c.github.io/preload/):
> This is a work in progress and may change without any notices.
> 
> Implementers SHOULD be aware that this document is not stable.

Of course, on the other hand, if it doesn't break older browsers without support, then I suppose it won't hurt anything. 👍

Reply:
hey @tarkatronic  😄 , the basic definition already exists in the spec https://html.spec.whatwg.org/multipage/links.html#link-type-preload. Further details for preload is in Candidate Recommendation status.